swan-lake/development-tutorials/ballerina-persist/persist-cli-tool.md

```diff
@@ -154,26 +166,23 @@ Behaviour of the `generate` command,
 - User should invoke the command within a Ballerina project
 - The model definition file should contain the `persist` module import (`import ballerina/persist as _;`)
 - The model definition file should contain at least one entity
-- If the user invokes the command twice, it will not fail. It will generate the files once again.
+- If the user invokes the command twice, it will fail if the client's native dependency exists in the `Ballerina.toml` file and is different from the current version. The user should remove the persist native dependency from the `Ballerina.toml` configurations to generate the files again.
 
-## Generate the data model by introspecting an existing database [Experimental]
-
->**Info:** The support for introspection is currently an experimental feature, and its behavior may be subject to change in future releases. Also, the support for introspection is currently limited to the MySQL data store.
+## Generate the data model by introspecting an existing database
 
 The command below generates the data model for an existing database. The generated data model can be used to generate the client API for the existing database after verifying it.
 
 ```
 $ bal persist pull --datastore mysql --host localhost --port 3306 --user root --database db
 ```
 
-| Command Parameter |                                   Description                                    | Mandatory | Default Value |
-|:-----------------:|:--------------------------------------------------------------------------------:|:---------:|:-------------:|
-|    --datastore    | used to indicate the preferred data store. Currently, only 'mysql' is supported. |    No     |     mysql     |
-|      --host       |                        used to indicate the database host                        |    Yes    |     None      |
-|      --port       |                        used to indicate the database port                        |    No     |     3306      |
-|      --user       |                        used to indicate the database user                        |    Yes    |     None      |
-|    --database     |                        used to indicate the database name                        |    Yes    |     None      |
-
+| Command Parameter |                                               Description                                               | Mandatory | Default Value |
```

Comment:
Is this a Command Parameter or a Command Option?

Reply:
Fixed in [5e81976](https://github.com/ballerina-platform/ballerina-dev-website/pull/9246/commits/5e81976916a09b9206b6f718416408d5686ab873)